src/ResourceManager/ContainerInstance/Commands.ContainerInstance/Commands/GetAzureContainerGroupLogsCommand.cs

```diff
@@ -0,0 +1,55 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+using Microsoft.Azure.Management.ContainerInstance;
+
+namespace Microsoft.Azure.Commands.ContainerInstance
+{
+    /// <summary>
+    /// Get-AzureRmContainerGroupLogs
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, ContainerLogsNoun), OutputType(typeof(string))]
```

Comment:
@yolo3301 do not output a `string`. Depending on the size of this log, we should write it to a file and change the name of this cmdlet to `Export-AzureRmContainerGroupLog`. In that case, this cmdlet would return nothing.

Reply:
Changed to Export